content/blog/2020/01/2020-01-10-fosdem-is-coming.adoc

```diff
@@ -0,0 +1,29 @@
+---
+layout: post
+title: "FOSDEM 2020 is coming"
+tags:
+- fosdem
+- event
+- community
+- outreachy
+author: olblak
+---
+
+image::/images/post-images/fosdem[Fosdem, role=center]
+
+link:https://fosdem.org/2020/[FOSDEM] 2020 is coming and with it, a lot of great folks come in town.  
+It’s always a great moment to meet Jenkins community members, share stories and get inspired.
+I hope that this year will be as great as it always been and for that, we organize a few things
+
+== Things we'll do
+
+On the Thursday 30 of January, there will be two workshops one about link:https://www.eventbrite.com/e/jenkins-pipeline-fundamentals-training-tickets-87080214265[Jenkin Pipelines] lead by Mark Waite, and a second one about link:https://www.eventbrite.com/e/workshop-cloud-native-kubernetes-first-serverless-continuous-delivery-with-jenkins-x-kubernetes-and-tickets-87082627483[JenkinsX] by Viktor Farcic. We are asking to register in advance and ask a small fee to avoid no-show people.
```

Comment:
The fee is mainly to cover catering. I thing it would convene the message better than fighting no-shows

Reply:
As far as I know, there will be only snacks which means that it's quite expensive just to cover catering, so I think it would be better to just remove the 'noshow' thing

Reply:
"Catering and swag"?